src/views/LandingPageView/Tickets/Tickets.jsx

```diff
@@ -102,12 +102,12 @@ const Tickets = () => (
         </div>
       </Switcher>
 
-      <Center maxWidth='800px' gutters='12px' withText>
+      <Center maxWidth='900px' gutters='12px' withText>
         <PricesText>
           Tous les tarifs indiqués sont avant taxes. Les billets sont
           non-remboursables, mais peuvent être transférés à une autre personne
-          avant le début de l’événement.
-          <br /> Les tarifs sont en vigueur jusqu&apos;au 20 mars inclusivement.
+          avant le début de l’événement. Les tarifs sont en vigueur
+          jusqu&apos;au 20 mars inclusivement.
```

Comment:
Je l'aurais mis en gras ici
```suggestion
         <strong>jusqu&apos;au 20 mars inclusivement</strong>.
```